### DISCUSSION TITLE
Sites of Importance for Nature Conservation (SINCs)

### DISCUSSION THREAD

**Discussion by RichLimbrick** [digital-land/data-standards-backlog#61] [Planning consideration]
These are areas of land that are especially important for their wildlife. SINCs were on the backlog so I've started a discussion on this as we're looking at ours at the moment. @colmjude any feedback very welcome on this.

There are some other designations which are very similar or equivalent including [Local Wildlife Sites (LWS) on the backlog of planning considerations](https://considerations.planning-data.dev/planning-consideration/local-wildlife-sites).

We have been reviewing our current schema and have produced a draft (sorry for clumsy formatting).
Field | Description
-- | --
geometry | WKT geometry
point | point in centre of   geometry
reference | unique reference for   the record than does not change
name | name of the SINC site
site-number | a number or reference   used to refer to the site or the evidence base supporting its designation - does not need to be unique
grade | the grade which MUST   be one of "Metropolitan", "Borough (grade 1)",   "Borough (grade 2)", or "Local"
notes | any notes about the   site or the record entry
access | access to the site,   SHOULD be one of “Free public access (all or most of site)”, “Free public   access (part of site)”, “Free public access (limited times)”, “Paid public   access”, “Viewable from adjacent paths or roads only”, “No public access”
entry-date | date the record was   entered
start-date | date the designation   comes into effect
end-date | date the designation   ceases to have effect
documentation-url | link to a webpage   about the SINC
document-url | link to a document   about the SINC

We would be interested for feedback on this particularly on:

- Does it work with the general DLUHC approach and should we change anything or describe it differently?
- Do the suggested "**grade**" categories work across the country or do we need to add more?
- Do others record the type of "**access**" to the site, and are these suggestions appropriate categories?

**Comment by RichLimbrick** [comment]
Other terms commonly used for similar non-statutory designations around the UK are:
•	London: Site of Importance for Nature Conservation (SINC)
•	Other areas of England: SINC or Local Wildlife Site (LWS) or Site of Nature Conservation Importance (SNCI)
•	Scotland: Local Nature Conservation Site
•	Wales: SINC
There can be other variations too like Wildlife Site.

Can this schema support all these?

**Reply by psd** [reply]
Thank Rich, that's a really interesting proposal. I found more info on the Wikipedia page  [Site of Nature Conservation Interest
](https://en.wikipedia.org/wiki/Site_of_Nature_Conservation_Interest)

The fields pretty consistent with our other specifications. Some minor suggestions:

1. `site-number` could be just be `site` to reference a site dataset
2. `grade` should probably be `site-grade` or something more specific to this dataset
3. `access` could maybe be `site-accessibility`

**Reply by RichLimbrick** [reply]
Thanks Paul - I think those are all better field names - much clearer and relate to the data. I'll update to those on our dataset. I'll root around for some other comparable outside London and see if it's easily adaptable.

**Reply by colmjude** [reply]
Are we saying LWS are synonymous with SINCs or that LWS are one type of SINC or that LWS are sometimes considered to be the same as SINCs?

If they are synonymous we can combine the 2 considerations on the backlog to make that a bit clearer.

**Comment by colmjude** [comment]
This is great thanks @RichLimbrick.

A couple of initial thoughts.

I think the schema aligns nicely to the DLUHC approach, so that's great. 
`grade` and `access` seem to be category-style fields. If these have a predefined set of values as the description suggests we would make these “datasets” in their own right, similar to how we have [listed-building-grade](https://dluhc-datasets.planning-data.dev/dataset/listed-building-grade) or [tree-preservation-zone-type](https://dluhc-datasets.planning-data.dev/dataset/tree-preservation-zone-type).

Building on Paul’s point - if these fields have their own supporting dataset then we’d need to make sure the name works as a stand-alone thing. Maybe something like `sinc-grade` and `sinc-accessibility`, but that assumes these “categories” can apply to any sincs, is that the case?

Also, do you have an example record you can share so that we can see what you expect the output to be?

**Comment by colmjude** [comment]
### Understanding SINCs

We are working on a more exhaustive checklist of things we need to know and questions we need to answer, but as a starter for ten, it’d be useful to get some information on the following.

To help understand the planning needs around SINCs, it would be useful to understand:
* What SINCs are for?
* How are they used?
* What effect do they have on planning? Do they prohibit things? Do they trigger some other planning requirement? That sort of thing.

In terms of who and how this data might get produced we’d need to understand:
* Who makes/designates them?
* What’s the high-level process for designating a SINC?
* What artefacts are produced during this process?
* Once a SINC, can an area stop being a SINC? How?

**Comment by drewhardy77** [comment]
Another SINC discussion [here](https://github.com/digital-land/data-standards-backlog/discussions/254). Closing this thread to avoid confusion
